src/api/APIFactory.js

```diff
@@ -273,6 +273,14 @@ class APIFactory {
         }
     }
 
+    /**
+     *
+     * @param {Token} token - token used to make requests
+     */
+    updateToken(token: Token = ''): void {
```

Comment:
Several questions about this approach:
- Should we be mutating `this.options`? 
- Should this method be named `setToken`? 
- Should this method have a default parameter value at all? 
- Should the factory destroy any in-flight requests if the token changes?